src/comments.js

```diff
@@ -42,24 +42,44 @@ function formatComment(options) {
   return result;
 }
 
-module.exports.postCreateComment = async function (appName, appUrl) {
+function buildLinks(appName, appUrl, message) {
+  const links = [];
+
+  const owner = github.context.payload.repository.owner.login;
+  const repo = github.context.payload.repository.name;
+  const prNumber = parseInt(github.context.payload.number, 10);
+  const sha = github.context.sha;
+
+  if (owner && repo && prNumber && sha && message) {
+    const commitLink = `https://github.com/${owner}/${repo}/pull/${prNumber}/commits/${sha}`;
+    const commitText = `\`${sha.substring(0, 8)}\` ${message.split('\n')[0]}`;
+    links.push(`:rocket: **Deployed commit:** [${commitText}](${commitLink})`);
```

Comment:
Ooooh I was wondering why you were constructing the commit links by hand like this... and I just learned that these two links result in different pages!
```
https://github.com/readmeio/heroku-review-app-action/commit/b0814f6230e49037186869dc3a5ceddbf8e47863
https://github.com/readmeio/heroku-review-app-action/pull/4/commits/b0814f6230e49037186869dc3a5ceddbf8e47863
```
Cool with this approach, linking to the commit within the PR is a lot more useful 👍🏽